audio/midiparser_smf.h

```diff
@@ -39,15 +54,17 @@ class MidiParser_SMF : public MidiParser {
 	 * @param numTracks The number of type 1 tracks.
 	 * @param buffer Buffer which will contain the compressed type 0 track.
 	 * @param malformedPitchBends True if broken pitch bend events consisting
+	 * @param compressedTrackingMetadata Optional compressed tracking metadata
 	 * of just the command byte should be ignored. This is only useful for MIDI
 	 * data which has this specific problem.
 	 * @return The size of the compressed type 0 track in bytes.
 	 */
-	uint32 compressToType0(byte *tracks[], byte numTracks, byte *buffer, bool malformedPitchBends = false);
+	uint32 compressToType0(byte *tracks[], byte numTracks, byte *buffer, bool malformedPitchBends);
```

Comment:
Are these leftovers from an earlier version of the code?

Reply:
Reverted that